src/test/util/test-rig.ts

```diff
@@ -141,6 +142,26 @@ export class WireitTestRig extends FilesystemTestRig {
       WIREIT_CACHE: undefined,
       ACTIONS_CACHE_URL: undefined,
       ACTIONS_RUNTIME_TOKEN: undefined,
+      // In npm 6 (which ships with Node 14), "npm run" only includes the
+      // immediate package's "node_modules/.bin" directory in $PATH. Above npm
+      // 6, all of the parent ".bin" directories are included too.
```

Comment:
Does this also create additional considerations when using wireit with npm 6 with parent/child packages for external users?

Reply:
I don't think so, that's just how npm 6 worked for any installed binary. It would be the same if you tried to execute `tsc` or `eslint` or whatever from a child package. I think this npm change to `$PATH` was probably made as part of adding the workspaces feature.